### PR TITLE
Spanish locale

### PR DESCRIPTION
**Pull request by n4k1**
First version has an extra line

### CODE REVIEW

**Comment by fmayoral** [comment]
Do **NOT** push directly to master, create a new feature/[descriptive-branch-name] branch and push to it, then create the pull request from it.
Commits need to be squashed.

File: src/_locales/es-ES.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{
+  "audio-device-communications": "Sistema de comunicaciones por defecto",
+  "audio-device-default": "Dispositivo por defecto",
+  "audio-device-unknown": "Dispositivo Desconocido",
+
+  "button-test-ok": "Aceptar",
+  "button-text-close": "Cerrar",
+  "button-text-dont-tell-me-again": "No recordar más",
+  "button-text-lets-do-this": "Hagámoslo!!",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: Missing standard reverse exclamation sign for spanish

File: src/_locales/es-ES.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  "button-text-not-now": "Ahora mismo no",
+
+  "label-about": "Acerca de",
+  "label-desktop-settings": "Ajustes de escritorio",
+  "label-donate": "Donar",
+  "label-help": "Ayuda",
+  "label-issues": "Problemas",
+  "label-learn-more": "Aprender más",
+  "label-loading-devices": "Cargando Dispositivos...",
+  "label-pause-after-song": "Pausar después de esta canción",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: "canción" implies a musical genere that always include lyrics, please consider using "pista" which is a more generic name for a musical piece.

File: src/_locales/es-ES.json [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  "label-quit": "Salir",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: "Salir" might be confusing in the context of an application, please consider using "Cerrar"

File: src/_locales/es-ES.json [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  "label-show-lyrics": "Mostrar Letras (Beta)",
+  "label-unknown-artist": "Artista desconocido",
+  "label-unknown-song": "Canción desconocida ",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: "canción" implies a musical genere that always include lyrics, please consider using "pista" which is a more generic name for a musical piece.

File: src/_locales/es-ES.json [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+  "label-version": "Versión",
+
+  "lastfm-login-authorized": "Autorizado",
+  "lastfm-login-authorizing": "Autorización en progreso",
+  "lastfm-login-button-text": "Accediendo a Last.FM",
+  "lastfm-login-error": "Parece que has dejado el proceso de autorización, si es un error, prueba a autorizar de nuevo",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: consider replacing
* dejado = abandonado
* si es un error = si ésto es un error
* prueba a autorizar de nuevo = intenta autorizar nuevamente

File: src/_locales/es-ES.json [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+  "label-pause-after-song": "Pausar después de esta canción",
+  "label-quit": "Salir",
+  "label-show-lyrics": "Mostrar Letras (Beta)",
+  "label-unknown-artist": "Artista desconocido",
+  "label-unknown-song": "Canción desconocida ",
+  "label-version": "Versión",
+
+  "lastfm-login-authorized": "Autorizado",
+  "lastfm-login-authorizing": "Autorización en progreso",
+  "lastfm-login-button-text": "Accediendo a Last.FM",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: Consider "Iniciar sesión en Last.FM"

File: src/_locales/es-ES.json [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+  "lastfm-login-error": "Parece que has dejado el proceso de autorización, si es un error, prueba a autorizar de nuevo",
+  "lastfm-login-not-authorized": "No autorizado",
+  "lastfm-logout-button-text": "Saliendo de Last.FM",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: Consider "Cerrar sesión en Last.FM"

File: src/_locales/es-ES.json [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+  "lyrics-failed-message": "No hemos enconrado la letra de esta canción",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: Typo "enconrado" = "encontrado"
Again "canción" implies a musical genere that always include lyrics, please consider using "pista" which is a more generic name for a musical piece.

File: src/_locales/es-ES.json [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+  "lyrics-loading-message": "Cargando letra...",
+  "lyrics-no-song-message": "No hay canción sonando ahora",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: Again "canción" implies a musical genere that always include lyrics, please consider using "pista" which is a more generic name for a musical piece.

And btw "sonando" is a poor selection,  consider something like "Actualmente no hay una pista en reproducción"

File: src/_locales/es-ES.json [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+
+  "message-pausing-after-song": "Pausando después de esta canción.",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: consider replacing
"después" = "luego"
"canción" ="pista"

File: src/_locales/es-ES.json [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+  "message-pausing-after-song-button": "No pausar después de esta canción",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: consider replacing
"después" = "luego"
"canción" ="pista"

File: src/_locales/es-ES.json [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+  "message-uncaught-error": "Un error inadvertido ha ocurrido dentro de GPMDP.",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: "inadvertido" is not quite a right translation for uncaght, "inesperado" is a better choice, as an uncaught error is usually an unexpected one. Also "dentro de" is a bit wordy, "en" sounds better.

File: src/_locales/es-ES.json [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+  "message-uncaught-error-button": "Haz clic aquí para reportar un problema en GitHub",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: Consider "Clic aquí para reportar éste error en GitHub" or similar

File: src/_locales/es-ES.json [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+
+  "modal-confirmTray-title": "Aviso",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: Not a right translation, "Sólo un aviso" fits better here

File: src/_locales/es-ES.json [hunk 15]
```diff
@@ -0,0 +1,96 @@
...
+  "modal-confirmTray-content": "Cerrando la ventana principal (justo lo que has intentado) hará que minimice el reproductor a la bandeja del sistema. Si deseas cambiar este comportamiento puede ser cambiado en la ventana de \"Ajustes de escritorio\".",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: "Cerrar la ventana principal del reproductor (lo que acabas de intentar hacer) lo minimizará a la bandeja del sistema. Éste comportamiento se puede cambiar en la ventana de  \"Ajustes de Escritorio\"."
Is a better translation

File: src/_locales/es-ES.json [hunk 16]
```diff
@@ -0,0 +1,96 @@
...
+  "modal-confirmUpdate-title": "Actualización disponible",
+  "modal-confirmUpdate-content": "Hey! Parece que hay una nueva actualización para Google Play Music Desktop Player.  Esta actualización debe llevar muchas nuevas características, correcciones de errores y de cosas nuevas interesantes.  Recomendamos que actualices lo antes posible para disfrutar las nuevas características y sin bugs .<br />Esperamos que os guste la nueva actualización!",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: Consider "¡Hey!, parece que hay una nueva actualización para Google Play Music Desktop Player.  Ésta actualización incluye nuevas características, soluciones de problemas y cosas nuevas geniales. Recomendamos que lo actualices lo antes posible para una mejor experiencia con nuevas características.<br />¡Esperamos que te guste la nueva actualización!" as a more semantic/syntactic friendly alternative

File: src/_locales/es-ES.json [hunk 17]
```diff
@@ -0,0 +1,96 @@
...
+  "modal-confirmUninstall-title": "Esta es la realidad...",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: consider "Resulta que..." as a more accurate translation

File: src/_locales/es-ES.json [hunk 18]
```diff
@@ -0,0 +1,96 @@
...
+  "modal-confirmUninstall-content": "Se ha encontrado una versión anterior de GPMDP instalada en el sistema. Si acabas de actualizar es normal. Vamos a desinstalar la versión anterior.  Esto \"puede\" necesitar permisos de administrador, por favor concede esos permisos para que podamos limpiar nuestro caos.",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: mess != "caos", consider "lío"

File: src/_locales/es-ES.json [hunk 19]
```diff
@@ -0,0 +1,96 @@
...
+  "modal-goToURL-title": "Inserta una URL de Google Play Music para navegar:",
+  "modal-welcome-title": "Bienvenido a la Versión",
+  "modal-welcome-content": "Esperamos que os guste la nueva actualización!",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: "os" => "te" is a more friendly/cool way to say "you"

File: src/_locales/es-ES.json [hunk 20]
```diff
@@ -0,0 +1,96 @@
...
+  "settings-option-custom-theme-custom-color": "Color personalizado",
+  "settings-option-custom-theme-dark": "Oscuro",
+  "settings-option-custom-theme-light": "Claro",
+
+  "settings-option-enable-api": "Activar reproducción a través de API",
+  "settings-option-enable-api-details": "Esta opción activa el uso de GPMDP por otras aplicaciones.",
+  "settings-option-enable-system-borders": "Usar sistema de ventanas  de sistema operativo",
+  "settings-option-enable-system-borders-details": "Requiere reinicio",
+  "settings-option-enable-voice": "Activar controles por voz",
+  "settings-option-enable-voice-details": "Controles por voz para GPMDP, esta función es experimental",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: "esta" should be "ésta"

File: src/_locales/es-ES.json [hunk 21]
```diff
@@ -0,0 +1,96 @@
...
+
+  "settings-option-hotkey-explanation": "Las teclas de función multimedia continuaran funcionando a pesar de los cambios. Estos ajustes son adicionales a los que están por defecto.",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: "continuaran" should be "continuarán"

File: src/_locales/es-ES.json [hunk 22]
```diff
@@ -0,0 +1,96 @@
...
+  "settings-option-hotkey-explanation-2": "Usa la tecla <i>\"Escape\"</i> para reiniciar la tecla de acceso rápido.",
+
+  "settings-option-min-to-tray": "Minimizar en bandeja del sistema",
+
+  "settings-option-auto-launch": "Arrancar automáticamente al iniciar el sistema.",
+
+  "settings-option-mini-always-show": "Mostrar siempre la información de la canción",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: "canción" should be "pista"

File: src/_locales/es-ES.json [hunk 23]
```diff
@@ -0,0 +1,96 @@
...
+  "settings-option-mini-ontop": "Siempre visible",
+
+  "title-color-picker": "Selector de color",
+  "title-settings": "Ajustes",
+  "title-settings-audio": "Audio",
+  "title-settings-general": "General",
+  "title-settings-hotkeys": "Hotkeys",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: "Atajos"

File: src/_locales/es-ES.json [hunk 24]
```diff
@@ -0,0 +1,96 @@
+{
+  "audio-device-communications": "Sistema de comunicaciones por defecto",
+  "audio-device-default": "Dispositivo por defecto",
+  "audio-device-unknown": "Dispositivo Desconocido",
+
+  "button-test-ok": "Aceptar",
+  "button-text-close": "Cerrar",
+  "button-text-dont-tell-me-again": "No recordar más",
+  "button-text-lets-do-this": "Hagámoslo!!",
+  "button-text-lets-go": "Vamos!!",
```
Comment (fmayoral) [inline src/_locales/es-ES.json]: Missing standard reverse exclamation sign for spanish
 Add a line note

**Comment by fmayoral** [comment]
Needs some improvements as translations are not accurrate and might be confusing in the application context, squash commits, and move the spanish translation into a proper feature branch or locale branch as mentioned in [Contributing Guidelines](https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/blob/master/CONTRIBUTING.md)

**Comment by MarshallOfSound** [comment]
@fmayoral  @n4k1 Please rebase and I will merge
